packages/core/src/types.ts

```diff
@@ -352,6 +355,26 @@ export type NormalizedHonoOptions = {
   handlers?: string;
 };
 
+export type ZodOptions = {
+  strict?: {
```

Comment:
Although it is not required for this PR, I think it would be better if the guide updates are done together so that users can recognize them.

Reply:
You mean update the doc?

Reply:
Yes, It seems good to add the `output.override.zod` item.

https://orval.dev/reference/configuration/output#override

Reply:
I updated the doc to add the output reference

Reply:
Thanks. That's great !

Reply:
However, it seems like there is a formatting error again.